notebooks/predicting_queries/simple_intent_suggestion.ipynb

```diff
@@ -0,0 +1,570 @@
+    "  pageview_timestamp,\n",
+    "  ROW_NUMBER() OVER (PARTITION BY action.session_id ORDER BY pageview_timestamp DESC) as hit_n\n",
+    "  FROM (\n",
+    "   SELECT\n",
+    "   geoNetwork.country,\n",
+    "   geoNetwork.region,\n",
+    "   geoNetwork.metro,\n",
+    "  CONCAT(CAST(fullVisitorId AS STRING), CAST(visitId AS STRING)) AS session_id,\n",
+    "  string_agg(LOWER(hits.page.searchKeyword)) as search_terms,\n",
+    "  MIN(TIMESTAMP_SECONDS(visitStartTime+CAST(hits.time/1000 AS INT64))) as first_search_timestamp\n",
+    "\n",
+    "FROM\n",
+    "  `govuk-bigquery-analytics.87773428.ga_sessions_*`,\n",
+    "  UNNEST(hits) AS hits\n",
+    "WHERE\n",
+    "  _table_suffix BETWEEN FORMAT_DATE('%Y%m%d',DATE_SUB(CURRENT_DATE(), INTERVAL 2 DAY))\n",
+    "                AND FORMAT_DATE('%Y%m%d',DATE_SUB(CURRENT_DATE(), INTERVAL 1 DAY))\n",
+    "  AND hits.page.searchKeyword IS NOT NULL\n",
+    "   group by session_id, geoNetwork.region,geoNetwork.metro, geoNetwork.country\n",
+    "  ) AS action\n",
+    "  LEFT JOIN \n",
+    "  (SELECT \n",
+    "  CONCAT(CAST(fullVisitorId AS STRING), CAST(visitId AS STRING)) AS session_id,\n",
+    "  hits.page.pageTitle as pageTitle,\n",
+    "  hits.page.pagePath as pagePath,\n",
+    "  ROW_NUMBER() OVER (PARTITION BY CONCAT(CAST(fullVisitorId AS STRING), CAST(visitId AS STRING)) ORDER BY hits.time DESC) as last_question,\n",
+    "  TIMESTAMP_SECONDS(visitStartTime+CAST(hits.time/1000 AS INT64)) as pageview_timestamp,\n",
+    "  FROM\n",
+    "  `govuk-bigquery-analytics.87773428.ga_sessions_*` \n",
+    "  CROSS JOIN UNNEST(hits) AS hits\n",
+    "  WHERE \n",
+    "      _table_suffix BETWEEN FORMAT_DATE('%Y%m%d',DATE_SUB(CURRENT_DATE(), INTERVAL 3 DAY))\n",
+    "      AND FORMAT_DATE('%Y%m%d',DATE_SUB(CURRENT_DATE(), INTERVAL 1 DAY)) \n",
+    "      AND hits.type = \"PAGE\"\n",
+    "      AND hits.page.pagePath LIKE '/transition-check/questions?%'\n",
+    "  ) as viewedpages\n",
+    "  ON viewedpages.session_id = action.session_id\n",
+    "  where pageTitle is not null\n",
+    "  AND last_question=1\n",
+    "  AND pageview_timestamp < first_search_timestamp\"\"\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#acc_df = pandas_gbq.read_gbq(acc_query)\n",
+    "#acc_df.to_csv(PROCESSED_DATA+'/accounts.csv')\n",
+    "#Clean data stored in urls (annoying)\n",
+    "#acc_df['responses']= acc_df['pagePath'].map(lambda x:x.split('c[]'))\n",
+    "#acc_df = acc_df.explode('responses')\n",
+    "#acc_df['clean'] = acc_df['responses'].map(lambda x: x.split('&')[0])\n",
+    "#acc_df = acc_df[acc_df['clean']!='/transition-check/questions?page=1']\n",
+    "#acc_df = acc_df[acc_df['clean']!='/transition-check/questions?']\n",
+    "#acc_df['clean'] = acc_df['clean'].map(lambda x: x.replace('=',''))\n",
+    "\n",
+    "acc_df = pd.read_csv(PROJECT_DIR+'/data/processed/accounts.csv')"
```

Comment:
Think you need to change the line where you import the data otherwise the subsequent cells won't work.
```suggestion
    "#acc_df = pandas_gbq.read_gbq(acc_query)\n",
    "#acc_df.to_csv(PROCESSED_DATA+'/accounts.csv')\n",
    "acc_df = pd.read_csv(PROJECT_DIR+'/data/processed/accounts.csv')"
    "#Clean data stored in urls (annoying)\n",
    "acc_df['responses']= acc_df['pagePath'].map(lambda x:x.split('c[]'))\n",
    "acc_df = acc_df.explode('responses')\n",
    "acc_df['clean'] = acc_df['responses'].map(lambda x: x.split('&')[0])\n",
    "acc_df = acc_df[acc_df['clean']!='/transition-check/questions?page=1']\n",
    "acc_df = acc_df[acc_df['clean']!='/transition-check/questions?']\n",
    "acc_df['clean'] = acc_df['clean'].map(lambda x: x.replace('=',''))\n",
```

Reply:
yep!